lib/sso_users_api/manager_job.rb

```diff
@@ -8,19 +8,24 @@ def self.delay_amount
       2
     end
 
-    def perform(id, class_name, count = 0)
+    def perform(id, class_name, on_success_callback_job = {}, count = 0)
 
       user = class_name.constantize.find(id)
       SsoUsersApi::Manager.new(user).call
```

Comment:
Is it possible to get api definition of response of SSO call, if it fails with an error then we can not create evo admin?  Check on status of the call if it is 2xx then proceed to create admin on evo?

Reply:
See if Ram can provide you that